config/config.yaml

```diff
@@ -162,8 +162,7 @@ defaults:
     rg: hcp-underlay-{{ .ctx.regionShort }}-imagesync
     acrRG: global
     environmentName: aro-hcp-image-sync
-    # secretsToSync: "dev_global_imagesync_bearer-secret.enc;dev_global_imagesync_component-sync-pull-secret.enc;"
-    secretsToSync: ""
+    secretsToSync: "dev/global/arohcp-imagesync-dev/bearer-secret.enc,dev/global/arohcp-imagesync-dev/component-sync-pull-secret.enc"
```

Comment:
how about one more simplification
`secretsToSync: "dev/global/arohcp-imagesync-dev"`

the script can find the files in the directory itself

Reply:
Hm, I'd prefer explicit configuration, i.e. to control secret rotation

Reply:
hmm. if you don't want a secret to be updated on the target, don't update it in the source.

please elaborate on how a secrets get updated in the ARO-HCP repo

Reply:
let's discuss the update process for secrets first

Reply:
Two ways to update:

- Create a new secret with a suffix/prefix (i.e. v2), run the pipeline; update the secret reference to use new version
or
- Update the secret and run the pipeline; perhaps recycle pods if needed